typescript/src/schema/proto.ts

```diff
@@ -348,9 +348,9 @@ function augmentMethod(
       }
     }
   }
-  if (method.inputType && parameters.allMessages[method.inputType]?.field) {
+  if (method.inputType && parameters.localMessages[method.inputType]?.field) {
     const paramComment: Comment[] = [];
-    const inputType = parameters.allMessages[method.inputType!];
+    const inputType = parameters.localMessages[method.inputType!];
```

Comment:
Thank you for fixing this!  I feel confused now. Was the message missing in `allMessages`? This is really weird, since `allMessages` should, by definition, contain all messages.

Reply:
I think the empty request params in the comments are extra ones, they are added by `allMessages`, but not exist in `localMessages`